packages/expo-cli/src/commands/init.ts

```diff
@@ -161,53 +163,194 @@ async function action(projectDir: string, command: Command) {
     packageManager = 'yarn';
-    log('Using Yarn to install packages. You can pass --npm to use npm instead.');
+    log.newLine();
+    log('🧶 Using Yarn to install packages. You can pass --npm to use npm instead.');
+    log.newLine();
   } else {
     packageManager = 'npm';
-    log('Using npm to install packages. You can pass --yarn to use Yarn instead.');
+    log.newLine();
+    log('📦 Using npm to install packages. You can pass --yarn to use Yarn instead.');
+    log.newLine();
   }
 
-  let projectPath = await Exp.extractAndInitializeTemplateApp(
-    templateSpec,
-    path.join(
-      parentDir,
-      dirName || ('expo' in initialConfig ? initialConfig.expo.slug : initialConfig.name)
-    ),
-    packageManager,
-    initialConfig
-  );
+  let extractTemplateStep = logNewSection('Downloading and extracting project files.');
+  let projectPath;
+  try {
+    projectPath = await Exp.extractAndPrepareTemplateAppAsync(
+      templateSpec,
+      path.join(
+        parentDir,
+        dirName || ('expo' in initialConfig ? initialConfig.expo.slug : initialConfig.name)
+      ),
+      initialConfig
+    );
+    extractTemplateStep.succeed('Downloaded and extracted project files.');
+  } catch (e) {
+    extractTemplateStep.fail(
+      'Something went wrong in downloading and extracting the project files.'
+    );
+    throw e;
+  }
+
+  // for now, we will just init a git repo if they have git installed and the
+  // project is not inside an existing git tree, and do it silently. we should
+  // at some point check if git is installed and actually bail out if not, because
+  // npm install will fail with a confusing error if so.
+  try {
+    // check if git is installed
+    // check if inside git repo
+    await Exp.initGitRepoAsync(projectPath, { silent: true });
+  } catch (_) {
+    // todo: check if git is installed, bail out
+  }
+
+  let installJsDepsStep = logNewSection('Installing JavaScript dependencies.');
+  try {
+    await Exp.installDependenciesAsync(projectPath, packageManager, { silent: true });
+    installJsDepsStep.succeed('Installed JavaScript dependencies.');
+  } catch (e) {
+    installJsDepsStep.fail('Something when wrong installing JavaScript dependencies.');
+  }
 
   let cdPath = path.relative(process.cwd(), projectPath);
   if (cdPath.length > projectPath.length) {
     cdPath = projectPath;
   }
-  log.nested(`\nYour project is ready at ${projectPath}`);
-  log.nested('');
   if (isBare) {
+    let podsInstalled = false;
+    try {
+      podsInstalled = await installPodsAsync(projectPath);
+    } catch (_) {}
+
+    log.newLine();
+    logProjectReady({ cdPath, packageManager, workflow: 'bare' });
+    if (!podsInstalled && process.platform === 'darwin') {
+      log.newLine();
+      log.nested(
+        `⚠️  Before running your app on iOS, make sure you have CocoaPods installed and initialize the project:`
+      );
+      log.nested('');
+      log.nested(`  cd ${cdPath || '.'}/ios`);
```

Comment:
```suggestion
      log.nested(`  cd ${cdPath ?? '.'}/ios`);
```
Also when is `cdPath` not defined?

Reply:
if you init in the same directory it seems, `expo init .` i guess